.changeset/giant-avocados-cover.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/eth-beacon-adapter': major
+---
+
+Changed API method from GET to POST for getting balances. RPC_URLs need to be updated to use POST method.
```

Comment:
Since these changesets are really the release notes, I think we need to add something here like
> strict requirement on Eth Beacon Node API >= v2.5.0

otherwise, this EA will fail with incompatible RPCs.

Reply:
Thinking about it, we might actually want to add this info into the README somewhere (known issues?)